.github/workflows/cicd.yml

```diff
@@ -374,6 +374,41 @@ jobs:
       fault_injection: true
       leak_detection: true
 
+  # Run multi-threaded stress test with 'restart extension' disabled (default behavior)
+  # against the kernel mode eBPF sub-system.
+  xyzzy_km_mt_stress_test:
+    needs: regular
+    if: github.repository == 'microsoft/ebpf-for-windows'
+    uses: ./.github/workflows/reusable-test.yml
+    with:
+      name: xyzzy_kernel_mode_multi_threaded_stress
+      pre_test: .\setup_ebpf_cicd_tests.ps1
```

Comment:
when published, this PR should use a stripped-down version of this setup script - no need to run these functions:
Get-RegressionTestArtifacts
Get-Duonic
Initialize-NetworkInterfacesOnVMs

Reply:
Fixed.